include/constantine/protocols/ethereum_bls_signatures.h

```diff
@@ -50,6 +50,13 @@ static const char* ctt_eth_bls_status_to_string(ctt_eth_bls_status status) {
   return "cttEthBls_InvalidStatusCode";
 }
 
+// Wrapper of the View[T] Nim type for the common case of View[byte]
+//
+// type View*[byte] = object # with T = byte
+//  data: ptr UncheckedArray[byte] # 8 bytes
+//  len*: int                      # 8 bytes (Nim `int` is a 64bit int type)
+typedef struct { byte* data; size_t len; } ByteView;
```

Comment:
```suggestion
typedef struct { byte* data; size_t len; } ctt_span;
```

C++20 and C# have settled to call this abstraction a span, see https://www.cppstories.com/2023/span-cpp20/, and original proposal: https://open-std.org/JTC1/SC22/WG21/docs/papers/2016/p0122r1.pdf

I think we should follow suit.

Reply:
If we 'embrace' it like this, I guess we could consider moving it to `core/datatypes.h` as well?

Reply:
Sounds good to my. Yes we can call it `ctt_span_bytes`, but I think it's fine for now, it's only used for arrays of messages. For the others we just have pointer+length.